app/src/main/java/org/jellyfin/androidtv/util/apiclient/ReportingHelper.kt

```diff
@@ -87,7 +89,9 @@ class ReportingHelper(
 
 		lifecycleOwner.lifecycleScope.launch {
 			Timber.d("Reporting ${item.name} playback progress at $position")
-			api.playStateApi.reportPlaybackProgress(info)
+			runCatching {
+				api.playStateApi.reportPlaybackProgress(info)
+			}.onFailure { error -> Timber.w(error, "Failed to report playback progress") }
```

Comment:
This one isn't as excited as the others

Reply:
Correct, if the start/stop fails it can screw with activity / playback reporting etc. A missed progress update can be easily recovered with the next update.